apps/dashboard/src/components/integrations/components/integration-settings.tsx

```diff
@@ -139,7 +142,49 @@ export function IntegrationConfiguration({
           </AccordionItem>
         </Accordion>
 
-        <Separator className="mb-0 mt-0" />
+        {isDemo ? (
+          <div className="p-3">
+            <InlineToast
+              variant={'warning'}
+              title="Demo Integration"
+              description={`This is a demo ${provider?.channel.toLowerCase()} integration intended for testing purposes only. It is limited to 300 notifications per month.${
+                provider?.channel === ChannelTypeEnum.EMAIL
+                  ? ' You can only send emails from it to the email address you are logged in with.'
+                  : ''
+              }`}
+            />
+          </div>
+        ) : (
+          provider.configurations && (
```

Comment:
@djabarovgeorge I believe we should feature flag this?

Reply:
sound good to me, i will add one